### PR TITLE
docs: add user guide pages

### PR DESCRIPTION
**Pull request by Wolbyte**
This pull request aims to add a new `User Guide` section with pages covering the basics of nixvim installation and usage, as well as some FAQs and tips.

@GaetanLepage is the one helping to document those topics.

### CODE REVIEW

**Review by trueNAHO — APPROVED** [review]

File: docs/pages/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# FAQ
+
+## Bugs
+
+### _The option `home` does not exist
```
Comment (trueNAHO) [inline docs/pages/FAQ.md:5]:
```suggestion
### The option `home` does not exist
```

**Review by trueNAHO — CHANGES_REQUESTED** [review]

File: docs/pages/FAQ.md (hunk 1 above)
Comment (GaetanLepage) [reply docs/pages/FAQ.md]: Thanks !

**Comment by MattSturgeon** [comment]
I believe this was superseded by #1121 and can be closed?

**Comment by traxys** [comment]
I think yes

**Pull request closed without merging**